tests/Support/SupportNumberTest.php

```diff
@@ -208,6 +212,79 @@ public function testToHuman()
         $this->assertSame('-1 thousand quadrillion', Number::forHumans(-1000000000000000000));
     }
 
+    public function testToHumansWithLocalization()
+    {
+        $this->mockTranslator();
+
+        app('translator')->setLoaded([
+            '*' => [
+                '*' => [
+                    'sv' => [
+                        'thousand' => 'tusen',
+                        'million' => 'miljon',
+                        'billion' => 'miljard',
+                        'trillion' => 'biljon',
+                        'quadrillion' => 'biljard',
+                    ],
+                ],
+            ],
+        ]);
+
+        $this->assertSame('1 tusen', Number::forHumans(1000, locale: 'sv'));
+        $this->assertSame('10 tusen', Number::forHumans(10000, locale: 'sv'));
+        $this->assertSame('1 miljon', Number::forHumans(1000000, locale: 'sv'));
+        $this->assertSame('1 miljard', Number::forHumans(1000000000, locale: 'sv'));
+        $this->assertSame('1 biljon', Number::forHumans(1000000000000, locale: 'sv'));
+        $this->assertSame('1 biljard', Number::forHumans(1000000000000000, locale: 'sv'));
+        $this->assertSame('1 tusen biljard', Number::forHumans(1000000000000000000, locale: 'sv'));
+
+        $this->assertSame('-1 tusen', Number::forHumans(-1000, locale: 'sv'));
+        $this->assertSame('-1 miljon', Number::forHumans(-1000000, locale: 'sv'));
+        $this->assertSame('-1 miljard', Number::forHumans(-1000000000, locale: 'sv'));
+
+        $this->assertSame('1,23 tusen', Number::forHumans(1234, locale: 'sv', precision: 2));
+        $this->assertSame('1,2 tusen', Number::forHumans(1234, locale: 'sv', maxPrecision: 1));
+        $this->assertSame('1,23 miljon', Number::forHumans(1234567, locale: 'sv', precision: 2));
+    }
+
+    public function testToHumansWithPluralizedLocalization()
```

Comment:
I just want to preface this with saying that language is complex. Like really complex. It's doubtful that a relatively short and simple method can contain all the edge cases needed for full support of something as complex as this. I totally understand if this is not something that's worth merging, but I wanted to try it out so here it is. We can always revert the commit.